src/adapters/methods/reft.py

```diff
@@ -23,7 +23,7 @@ def __init__(
         self.orthogonal = orthogonal
         self.learned_source = nn.Linear(in_dim, r_dim, bias=True)
 
-        projection = nn.Linear(in_dim, r_dim)
+        projection = nn.Linear(in_dim, r_dim, bias=False)
         if orthogonal:
             nn.init.orthogonal_(projection.weight)
```

Comment:
nit: we might not need this as registering line below might trigger this call internally: https://github.com/pytorch/pytorch/blob/main/torch/nn/utils/parametrize.py#L421

i am also removing this line in the `pyvene` library -- kudos to `peft` library PR: https://github.com/huggingface/peft/pull/1657#discussion_r1570154894